Xamarin.Build.AsyncTask/Xamarin.Build.AsyncTask.csproj

```diff
@@ -32,7 +32,10 @@
     <PackageReference Include="Microsoft.Build.Tasks.Core" Version="15.6.82" />
     <PackageReference Include="Microsoft.Build.Utilities.Core" Version="15.6.82" />
 
-    <PackageReference Include="MSBuilder.ThisAssembly.Metadata" Version="0.1.4" PrivateAssets="all" />
+    <PackageReference Include="ThisAssembly.Metadata" Version="1.4.0">
```

Comment:
This package does seem OK, and doesn't use `SponsorLink`:

https://www.nuget.org/packages/ThisAssembly.Metadata/1.4.0#dependencies-body-tab

Reply:
Old package is deprecated anyway:
https://www.nuget.org/packages/MSBuilder.ThisAssembly.Metadata

Reply:
The Deprecation is why I wanted to switch :)